src/evict.c

```diff
@@ -708,3 +708,16 @@ int performEvictions(void) {
     return result;
 }
 
+int performEvictions(void) {
+    int result = doPerformEvictions();
+
+    if (result == EVICT_RUNNING) {
+        if (server.stat_last_eviction_exceeded_time == 0) {
```

Comment:
Not required under single line {}.

Reply:
I'm confused whther `{} needed. And space like
```
f(a,b,c)
f(a, b, c)
a+b
a + b
```
Some codes have space and some not.

Reply:
In the old code many of them are without spaces, the back style is basically with spaces, but if you modify the old code, I tend not to change the original style.